boards/nucleo_wl55jc.json

```diff
@@ -0,0 +1,46 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32WL55JC1",
+    "f_cpu": "48000000L",
+    "mcu": "stm32wl55jc",
+    "product_line": "STM32WL55xx",
+    "variant": "STM32WL55JCx/WL55JC(1-2)"
+  },
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32WL55JC",
+    "onboard_tools": [
+      "stlink"
+    ],
+    "openocd_target": "stm32wlx",
+    "svd_path": "STM32WL5x_CM4.svd"
+  },
+  "frameworks": [
+    "arduino",
+    "cmsis",
+    "mbed",
+    "stm32cube",
+    "libopencm3",
+    "zephyr"
+  ],
+  "platforms": ["ststm32"],
```

Comment:
Seems redundant to me. It's set automatically if the board in located in the platform.

Reply:
I see. I saw this line in one of the json files I used for reference, guess I just assumed it had to be there.